Lab_4.ipynb

```diff
@@ -807,8 +807,16 @@
    "id": "3746fba1-b7d1-4416-b197-7bbd250bf70d",
    "metadata": {},
    "source": [
-    "We should get different pdfs because we are looking for different kinds of signals. I "
```

Comment:
Ok this is the only thing I can leave a comment on, but the problem listing is wrong. Above you've pasted the text for problem 1's part 4, not problem 2's.